### DISCUSSION TITLE
Feature Request:  In single file mode, is it possible to create notes by month or year?

### DISCUSSION THREAD

**Discussion by i-iooi-i** [Quorafind/Obsidian-Thino#649] [Ideas 想法]
### Feature Requested | 请求的功能

有个担忧，若是随着笔记内容的增多，一个文件里有几千条笔记或者更多笔记的时候，会比较不方便。
假设在同步过程中丢失了这1个文件，损失还是很大的。

若是可以增加自定义单文件创建周期的功能，比如按照月创建，或者年自动创建，这样担心文件丢失的心智负担会减小。

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
你可以把这个问题转移到讨论中去
